relay-server/src/services/store.rs

```diff
@@ -1269,7 +1309,7 @@ struct SpanKafkaMessage<'a> {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     segment_id: Option<&'a str>,
     #[serde(default, skip_serializing_if = "Option::is_none")]
-    sentry_tags: Option<&'a RawValue>,
+    sentry_tags: Option<BTreeMap<&'a str, &'a str>>,
```

Comment:
Remember special characters.

```suggestion
    sentry_tags: Option<BTreeMap<&'str, String>>,
```